### ISSUE TITLE
tree: draw.clade doesn't work for rectangular layout

### ISSUE THREAD

**Issue by ArtPoon** [ArtPoon/ggfree#26]
This is a tree I reconstructed from some SARS-CoV-2 genomes:
```R
> phy <- read.tree('~/git/covizu/data/timetree.nwk')
> phy

Phylogenetic tree with 263 tips and 177 internal nodes.

Tip labels:
  B.6.1, B.5, B.17, B.4, B.4.1, B.4.2, ...
Node labels:
  NODE_0000002, NODE_0000215, NODE_0000216, NODE_0000217, NODE_0000222, NODE_0000223, ...

Rooted; includes branch lengths.
```

This works fine for unrooted tree layout:
```R
> tips <- c("B.5", 'B.2.1', 'B.9')
> L <- tree.layout(phy, 'u')
> plot(L)
> draw.clade(L, tips, lwd=2)
```
<img width="550" alt="Screen Shot 2020-11-17 at 10 40 38 PM" src="https://user-images.githubusercontent.com/1109328/99480380-f2dc8880-2925-11eb-950d-c69a1133cb68.png">

But it messes up for rect layout:
```R
> L <- tree.layout(phy, 'r')
Warning message:
In as.phyloData(phy, unscaled) :
  as.phyloData ignoring attribute root.edge of length 1
> plot(L)
> draw.clade(L, tips, lwd=2)
```
<img width="525" alt="Screen Shot 2020-11-17 at 10 41 37 PM" src="https://user-images.githubusercontent.com/1109328/99480450-1a335580-2926-11eb-9d62-0b38983da064.png">

**Comment by ArtPoon** [comment]
Probably has to do with reversal of `parent` and `child` ordering with different placement of "root".

**Comment by ArtPoon** [comment]
This appears to be fixed with resolution of #30 
```R
require(ggfree)
set.seed(1)
phy <- rtree(20)
L <- tree.layout(phy, 'u')
plot(L)
draw.clade(L, tips=c('t10', 't20', 't7'), lwd=1.5)
```
<img src="https://user-images.githubusercontent.com/1109328/201427748-6d50c0e1-dee1-4208-9986-57bd800aa4be.png" width="500"/>

```R
L <- tree.layout(phy, 'r')
plot(L)
draw.clade(L, tips=c('t10', 't20', 't7'), lwd=1.5)
```
<img src="https://user-images.githubusercontent.com/1109328/201427772-72fe612d-1740-4aa5-ace0-64b09cfc2c01.png" width="500"/>

However this function is not drawing vertical line segments

**Comment by ArtPoon** [comment]
Got this working for circular layouts too:
```R
L <- tree.layout(phy, 'o')
plot(L)
draw.clade(L, tips=c('t10', 't20', 't7'), lwd=1.5)
```
<img width="500" src="https://user-images.githubusercontent.com/1109328/201433139-5b405451-6ad5-41db-a032-b06d880cbb6d.png"/>
